blockchain/chain.go

```diff
@@ -1687,21 +1693,22 @@ func New(config *Config) (*BlockChain, error) {
 		return nil, AssertError("blockchain.New timesource is nil")
 	}
 
-	// Generate a checkpoint by height map from the provided checkpoints.
-	params := config.ChainParams
+	// Create checkpoint variables. Leave empty when disabled.
```

Comment:
Yeah I changed it first to not only make the map there but also the slice. Forgot to change the comment back I guess.